ert3/config/_parameters_config.py

```diff
@@ -29,11 +29,14 @@ def _ensure_valid_name(name: str) -> str:
     if not name:
         raise ValueError("Names cannot be of zero length")
 
-    if not all(c.isalpha() or c == "_" for c in name):
+    if not all(c.isalnum() or c == "_" for c in name):
         raise ValueError(
-            "Names are expected to only contain characters and `_`, was: {name}"
+            "Names must consist of only characters, numbers " f"and `_`, was: {name}"
         )
 
+    if not (name[0].isalpha() or name[0] == "_"):
```

Comment:
```suggestion
    if not name[0].isalpha():
```

I guess this statement is only a hardening of the above condition for the first character... No need to repeat parts of the general requirement as well :)

Reply:
Right! Fixed to `.isdigit()`, thus allowing variables to start with `_` which was allowed before this PR.

Reply:
My argument and suggestion did not align! Sorry about that... Broken logics!

I suggest we move with the strictest rule and only allow `isalpha` for the first character as suggested above :)